docusaurus/docs/how-to-guides/app-plugins/implement-rbac-in-app-plugins.md

```diff
@@ -65,6 +85,116 @@ For example, in the above example, users with the `Viewer` role will automatical
 
 When defining roles, ensure each role is clearly differentiated with unique permissions to avoid conflicts and unintended access. It's best to follow the principle of least privilege, assigning the minimum permissions necessary for the tasks.
 
+## Actionsets, extend actions with permissions
+
+The action set system provides a clean way for plugins to extend Grafana's existing permission model without requiring users to understand or manually configure individual plugin actions. 
+Action Sets are a powerful feature that allows you to define **named collections of actions** that can be referenced as a single unit and used to extend existing roles. 
```

Comment:
I don't think it's accurate to say that action sets allow extending existing roles - they extend what it means to have View, Edit or Admin access to a folder.